feat: USB v3 migration guide (#1100)
edgexfoundry/edgex-docs: docs_src/microservices/device/V3Migration.md

This section is specific to changes made only to **Device USB Camera**

See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details applicable to all EdgeX services and [All Device Services](#all-device-services) section above for details applicable to all EdgeX device services.

#### Configuration

All USB camera rtsp streams need authentication by default. To properly configure credentials for the stream, follow the below instructions.

> 1. Set the credentials in the `configuration.yaml` 

Reviewer: It would be more helpful if `configuration.yml` points to the actual location.

    !!! example - "Snippet from configuration.yaml"
        ```yaml
        Writable:
            LogLevel: "INFO"
            InsecureSecrets:
              rtspauth:
                SecretName: rtspauth
                SecretData:
                  username: "<user>"
                  password: "<pass>"
        ```
    
    To use curl commands to set these values, see [here](./supported/device-usb-camera/walkthrough/deployment.md#add-credentials-for-the-rtsp-stream)

    

> 1. You can configure the authentication server to run from a different port by editing the externalAuthenticationURL value in the Dockerfile. To disable authentication entirely, comment out the externalAuthenticationURL line in the Dockerfile.

Reviewer: Same as above `Dockerfile` pointing to the actual location will be more useful for the user.

This section is specific to changes made only to **Device USB Camera**

See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details applicable to all EdgeX services and [All Device Services](#all-device-services) section above for details applicable to all EdgeX device services.

#### Configuration

> All USB camera rtsp streams need authentication by default. To properly configure credentials for the stream, follow the below instructions.

Reviewer 2: Would it make more sense to only have this link to the official documentation to avoid having to maintain 2 copies of it?

ie. this migration guide will merely tell/warn the users about the change, and then link to the documentation regarding it.

This section is specific to changes made only to **Device USB Camera**

See [Top Level V3 Migration Guide](../../../V3TopLevelMigration) for details applicable to all EdgeX services and [All Device Services](#all-device-services) section above for details applicable to all EdgeX device services.

#### Configuration

> All USB camera rtsp streams need authentication by default. To properly configure credentials for the stream see these [instructions](./supported/device-usb-camera/supplementary-info/advanced-options.md#rtsp-authentication). To see how to use this feature, see [here](./supported/device-usb-camera/walkthrough/deployment.md#add-credentials-for-the-rtsp-stream)

Reviewer: ```suggestion
All USB camera rtsp streams need authentication by default. To properly configure credentials for the stream refer [here](./supported/device-usb-camera/walkthrough/deployment.md#add-credentials-for-the-rtsp-stream).
To learn more about rtsp authentication refer [here](./supported/device-usb-camera/supplementary-info/advanced-options.md#rtsp-authentication).
```
I think rearranging of links might make better sense.
